app/controllers/planning_applications/review/pre_commencement_conditions_controller.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+module PlanningApplications::Review
+  class PreCommencementConditionsController < AuthenticationController
+    include CommitMatchable
+    include PlanningApplicationAssessable
+
+    before_action :set_planning_application
+    before_action :ensure_planning_application_is_validated
+    before_action :ensure_user_is_reviewer
+
+    def show
+    end
```

Comment:
For consistency should we add?
```ruby
respond_to do |format|
  format.html
end
```

Reply:
I wasn't sure how necessary this was, but I think it's better to be explicit, yes.